### PR TITLE
Merge hackathon results from team "byte brownies" on machine Studer SL20

### PR DESCRIPTION
**Pull request by cwtimobarth**

### CODE REVIEW

File: 2nd_hackathon/byte-brownies-studer-sl20/mqtt-broker-image/mqtt-broker-arm32v6.tar
Comment (GoetzGoerisch) [inline 2nd_hackathon/byte-brownies-studer-sl20/mqtt-broker-image/mqtt-broker-arm32v6.tar:1]: @BigPersia do we need the mosquitto tar file here?

File: 2nd_hackathon/byte-brownies-studer-sl20/ua-server/package.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
+{
+  "name": "cw-opcua-umati-server",
+  "version": "0.0.1",
+  "description": "",
+  "author": "",
+  "private": true,
+  "license": "UNLICENSED",
```
Comment (GoetzGoerisch) [inline 2nd_hackathon/byte-brownies-studer-sl20/ua-server/package.json:7]: Could we apply MIT license here?

**Review by GoetzGoerisch — COMMENTED** [review]
@BigPersia please add your company email address to your GitHub account please to associate the commits with your user. Thank you!

File: 2nd_hackathon/byte-brownies-studer-sl20/mqtt-broker-image/mqtt-broker-arm32v6.tar
Comment (cwtimobarth) [reply 2nd_hackathon/byte-brownies-studer-sl20/mqtt-broker-image/mqtt-broker-arm32v6.tar:1]: This is the actual image we used to get the broker running on the WAGO PLC.

I think it is enough to provide a link in the readme to that image.

**Comment by BigPersia** [comment]
> @BigPersia please add your company email address to your GitHub account please to associate the commits with your user. Thank you!

I added my company email adress but the cla shows still pending

File: 2nd_hackathon/byte-brownies-studer-sl20/ua-server/package.json (hunk 1 above)
Comment (cwtimobarth) [reply 2nd_hackathon/byte-brownies-studer-sl20/ua-server/package.json:7]: I"ll update it.

File: 2nd_hackathon/byte-brownies-studer-sl20/mqtt-broker-image/mqtt-broker-arm32v6.tar
Comment (cwtimobarth) [reply 2nd_hackathon/byte-brownies-studer-sl20/mqtt-broker-image/mqtt-broker-arm32v6.tar:1]: Added mqtt broker section in server readme and removed actual broker image.

**Comment by cwtimobarth** [comment]
> Thanks @BigPersia!
> 
> > I added my company email adress but the cla shows still pending
> > CLA was stuck.
> 
> Waiting for @cwtimobarth to resolve my comments, Good to merge then.

@GoetzGoerisch I made the necessarz changes and resolved the open issues.

**Review by GoetzGoerisch — APPROVED** [review]

**Pull request merged**